webpack.config.js

```diff
@@ -1,4 +1,5 @@
 const path = require('path');
+const webpack = require('webpack');
```

Comment:
Should this just be an import of `ProvidePlugin` specifically?

Reply:
I would be OK with switching it to:
```suggestion
const ProvidePlugin = require('webpack').ProvidePlugin;
```
or
```suggestion
const {ProvidePlugin} = require('webpack');
```

...but since this plugin is built into `webpack` (unlike `uglify-webpack-plugin`) it can't just be `require('provide-webpack-plugin')` or anything like that.

I wrote it the way it is because that's the way the example code is written in the Webpack docs, but since there's only one thing currently coming from `webpack` I don't feel strongly about which one we use. Maybe I lean toward `const {ProvidePlugin}` a bit, now that I've written out the options. Thoughts?